### PR TITLE
Use proper helm value in CI clusters

### PR DESCRIPTION
**Pull request by nebril**
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/8973)

### CODE REVIEW

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-me-please

File: test/helpers/kubectl.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ var (
 	defaultHelmOptions = map[string]string{
 		"global.registry":               "k8s1:5000/cilium",
 		"agent.image":                   "cilium-dev",
-		"agent.tag":                     "latest",
+		"global.tag":                    "latest",
 		"operator.image":                "operator",
 		"operator.tag":                  "latest",
```
Comment (brb) [inline test/helpers/kubectl.go]: Nice catch! Shouldn't we get rid of `operator.tag` as the operator Chart is using the `global.tag` to define the image tag?

Comment (nebril) [reply test/helpers/kubectl.go]: whoah, didn't see this one. Will add this to this PR

**Comment by nebril** [comment]
test-me-please

**Review by brb — APPROVED** [review]

File: test/helpers/kubectl.go (hunk 1 above)
Comment (brb) [reply test/helpers/kubectl.go]: I think we should revisit all the options.

**Review by ianvernon — APPROVED** [review]

**Pull request merged**